deployer/integration_test.go

```diff
@@ -13,23 +14,17 @@ import (
 // Successful Tests
 ///////////////
 
-var goodReleases = []string{
```

Comment:
these should pass

Reply:
yep, they do. Just changed it from manual list to automatically fetching all files in the allowed folder